Tasks/Maven/testlib/tasklib-wrapper.ts

```diff
@@ -0,0 +1,158 @@
+// Redirects calls to vsts-task-lib to testlib if in a test environment.
```

Comment:
All files under Tasks/Maven/testlib/ and Tasks/Gradle/testlib/ are identical between those two tasks.